elastic-apm-agent/pom.xml

```diff
@@ -197,7 +197,7 @@
                         </goals>
                         <configuration>
                             <verbose>true</verbose>
-                            <failOnUnableToExtractRepoInfo>false</failOnUnableToExtractRepoInfo>
+                            <failOnUnableToExtractRepoInfo>true</failOnUnableToExtractRepoInfo>
```

Comment:
The default is `true`, thus we can probably remove the config option completely.

Reply:
Yes, but if and when we bump into issues with this plugin again, we won't have to look it up again